src/repositories/component.repository.ts

```
@@ -607,585 +156,99 @@ export class ComponentRepository {
-      MATCH (repo:Repository {id: '${escapedRepositoryNodeId}'})<-[:PART_OF]-(algo_component_node)
-      RETURN algo_component_node AS component, k_degree
-    `;
+  /**
+   * Get governing decisions for a component
+   */
+  async getGoverningItemsForComponent(
+    repositoryName: string,
+    componentId: string,
+    componentBranch: string,
+  ): Promise<Decision[]> {
+    return this.graphRepo.getGoverningItemsForComponent(repositoryName, componentId, componentBranch);
```

ESLint

[error] Replace `repositoryName,·componentId,·componentBranch` with `⏎······repositoryName,⏎······componentId,⏎······componentBranch,⏎····`

(prettier/prettier)